.github/workflows/ci.yml

```diff
@@ -4,7 +4,8 @@ on:
   push:
     branches: [master]
   pull_request:
-    branches: [master]
+    branches:
+      - '**'
 
 jobs:
   build:
```

Comment:
```suggestion
  build:
    name: '${{ matrix.testname }}'
```
The name I saw in the run is a bit difficult to parse. You can add the django-env or format it differently, but you get the gist.

Reply:
That would be neat. But I'm not sure it's worth creating a migration issue

Reply:
Won't all this changes fall in to one issue?